### PR TITLE
Added Timsort implementation in Python

### PR DESCRIPTION
**Pull request by m-levytskyi**
It has O(n*logn) time complexity for worst case and O(n) for best case scenarios.

### CODE REVIEW

File: Python/Tim sort.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+def timsort(array):
```
Comment (manisaiprasad) [inline Python/Tim sort.py:4]: Mention Time Complexity

Comment (m-levytskyi) [reply Python/Tim sort.py:4]: It has O(n*logn) time complexity for worst case and O(n) for best case scenarios.

Comment (manisaiprasad) [reply Python/Tim sort.py:4]: Add that in the program and commit

**Comment by manisaiprasad** [comment]
Thank you for your interest in open source @qwertymaden

**Pull request merged**